Increase web interface get-devices timeout to 10s.

Patch-Origin: Fedora 21

--- a/cgi-bin/admin.c
+++ b/cgi-bin/admin.c
@@ -1019,13 +1019,13 @@ do_am_printer(http_t *http,		/* I - HTTP
     }
 
    /*
-    * Scan for devices for up to 30 seconds...
+    * Scan for devices for up to 10 seconds...
     */
 
     fputs("DEBUG: Getting list of devices...\n", stderr);
 
     current_device = 0;
-    if (cupsGetDevices(http, 5, CUPS_INCLUDE_ALL, CUPS_EXCLUDE_NONE,
+    if (cupsGetDevices(http, 10, CUPS_INCLUDE_ALL, CUPS_EXCLUDE_NONE,
                        (cups_device_cb_t)choose_device_cb,
 		       (void *)title) == IPP_OK)
     {
